Problems/2447/ans_2447.py

```diff
@@ -0,0 +1,29 @@
+def save_pattern(N):
```

Comment:
review

Reply:
Review by JHP
- Debugging & Compile : Success (VS code, Python 3.9)
- Need to follow up the logic